### PR TITLE
6. Análisis de código estatico

### PR DESCRIPTION
**Pull request by cardel**
image

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  'parser': '@typescript-eslint/parser',
+  'parserOptions': {
+    'ecmaVersion': 'latest',
+    'sourceType': 'module',
+  },
+  'plugins': [
+    '@typescript-eslint',
+  ],
+  'rules': {
+  },
```
Comment (kliver98) [inline .eslintrc.js]: Remember step 6.3 you're left some content

**Review by kliver98 — COMMENTED** [review]
Fix merge conflicts

**Comment by cardel** [comment]
Done, conflicts solved

File: .eslintrc.js (hunk 1 above)
Comment (holgiosalos) [inline .eslintrc.js]: you need to configure these two options as specified in step 6.3: https://github.com/holgiosalos/cypress-training-doc#6-an%C3%A1lisis-de-c%C3%B3digo-estatico

Comment (cardel) [reply .eslintrc.js]: Fixed

**Comment by cardel** [comment]
Fixed .eslintrc

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**